frontend/src/actions/appLib.ts

```diff
@@ -63,3 +66,31 @@ export async function restart(packageId: string) : Promise<string> {
     throw new Error(res.message)
   return res.message
 }
+
+// off specific service/node
+export async function off(packageId: string) : Promise<string> {
```

Comment:
@bonhokage06  be precise function naming. name can have different meanings. Instead of using  off, use disableApp or use any descriptive names

Reply:
got it sir Joe. thank you